python/GameOfLife/Bipeds.py

```diff
@@ -35,17 +33,22 @@ def filter_generation(self, gen):
         # key, value
         hum_gen_list = []
         gen_mapping = {
-            'genx': range(1965, 1980),
-            'genz': range(1997, 2012),
-            'gena': range(2010, 2020),
-            'mill': range(1981, 1996)
+            'boomers': range(1955,1964),
+            'genx': range(1965, 1981),
+            'genz': range(1997, 2013),
+            'gena': range(2010, 2021),
+            'zoomers': range(1997,2022),
+            'mill': range(1981, 1997)
         }
-        for key, gen_range in gen_mapping.items():
-            if key == gen.lower():
-                for people in self.pop_list:
-                    if people.birthday in gen_range:
-                        hum_gen_list.append(people)
-                return hum_gen_list
+        gen_range = gen_mapping[gen.lower()]
+
+        if gen.lower() != gen_mapping:
```

Comment:
les tests ne roullaient toujours pas donc je suis allé voir et c'est à cause que l'on regarde pour une inégalité plustot que si la clef est à l'intérieur du dict. Remplace `!=` par `not in`